2022-01-04 02:05:36,484 INFO    MainThread:90663 [wandb_setup.py:_flush():71] setting env: {}
2022-01-04 02:05:36,484 INFO    MainThread:90663 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-04 02:05:36,484 INFO    MainThread:90663 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-04 02:05:36,485 INFO    MainThread:90663 [wandb_init.py:_log_setup():371] Logging user logs to /home/puppy/projects/BCI/wandb/run-20220104_020536-1v32ty81/logs/debug.log
2022-01-04 02:05:36,485 INFO    MainThread:90663 [wandb_init.py:_log_setup():372] Logging internal logs to /home/puppy/projects/BCI/wandb/run-20220104_020536-1v32ty81/logs/debug-internal.log
2022-01-04 02:05:36,485 INFO    MainThread:90663 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fd76ce6d250>
2022-01-04 02:05:36,486 INFO    MainThread:90663 [wandb_init.py:init():404] calling init triggers
2022-01-04 02:05:36,486 INFO    MainThread:90663 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-04 02:05:36,487 INFO    MainThread:90663 [wandb_init.py:init():460] starting backend
2022-01-04 02:05:36,487 INFO    MainThread:90663 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-04 02:05:36,523 INFO    MainThread:90663 [backend.py:ensure_launched():216] starting backend process...
2022-01-04 02:05:36,558 INFO    MainThread:90663 [backend.py:ensure_launched():221] started backend process with pid: 90757
2022-01-04 02:05:36,562 INFO    MainThread:90663 [wandb_init.py:init():469] backend started and connected
2022-01-04 02:05:36,775 INFO    MainThread:90663 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-01-04 02:05:36,779 INFO    MainThread:90663 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-01-04 02:05:36,779 INFO    MainThread:90663 [wandb_init.py:init():533] updated telemetry
2022-01-04 02:05:36,788 INFO    MainThread:90663 [wandb_init.py:init():563] communicating current version
2022-01-04 02:05:41,789 INFO    MainThread:90663 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-04 02:05:44,913 INFO    MainThread:90663 [wandb_init.py:init():606] starting run threads in backend
2022-01-04 02:05:49,929 INFO    MainThread:90663 [wandb_run.py:_console_start():1810] atexit reg
2022-01-04 02:05:49,931 INFO    MainThread:90663 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-01-04 02:05:49,932 INFO    MainThread:90663 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-01-04 02:05:49,935 INFO    MainThread:90663 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-04 02:05:49,936 INFO    MainThread:90663 [wandb_init.py:init():633] run started, returning control to user process
2022-01-04 02:05:51,067 INFO    MainThread:90663 [wandb_init.py:_pause_backend():284] pausing backend
2022-01-04 02:05:51,068 INFO    MainThread:90663 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-01-04 02:08:43,303 INFO    MainThread:90663 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-01-04 02:08:43,307 INFO    MainThread:90663 [wandb_run.py:_restore():1752] restore
2022-01-04 02:08:43,679 INFO    MainThread:90663 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  total_bytes: 711
}

2022-01-04 02:08:43,990 INFO    MainThread:90663 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  total_bytes: 711
}

2022-01-04 02:08:44,150 INFO    MainThread:90663 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  total_bytes: 8279
}

2022-01-04 02:08:44,256 INFO    MainThread:90663 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  total_bytes: 8279
}

2022-01-04 02:08:44,360 INFO    MainThread:90663 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  total_bytes: 8279
}

2022-01-04 02:08:44,508 INFO    MainThread:90663 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  total_bytes: 8279
}

2022-01-04 02:08:44,612 INFO    MainThread:90663 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  total_bytes: 8279
}

2022-01-04 02:08:44,716 INFO    MainThread:90663 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  total_bytes: 8279
}

2022-01-04 02:08:44,821 INFO    MainThread:90663 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  total_bytes: 8279
}

2022-01-04 02:08:44,925 INFO    MainThread:90663 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  total_bytes: 8279
}

2022-01-04 02:08:45,029 INFO    MainThread:90663 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  total_bytes: 8279
}

2022-01-04 02:08:45,133 INFO    MainThread:90663 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  total_bytes: 8279
}

2022-01-04 02:08:45,237 INFO    MainThread:90663 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  total_bytes: 8279
}

2022-01-04 02:08:45,342 INFO    MainThread:90663 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  total_bytes: 8279
}

2022-01-04 02:08:45,448 INFO    MainThread:90663 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  total_bytes: 8279
}

2022-01-04 02:08:45,552 INFO    MainThread:90663 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  total_bytes: 8279
}

2022-01-04 02:08:45,656 INFO    MainThread:90663 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  total_bytes: 8279
}

2022-01-04 02:08:45,760 INFO    MainThread:90663 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  total_bytes: 8279
}

